openedx_webhooks/github_views.py

```diff
@@ -46,29 +49,34 @@ def pull_request():
         return "PONG"
 
     pr = event["pull_request"]
+    PR_NUMBER = pr['number']
     repo = pr["base"]["repo"]["full_name"].decode('utf-8')
     action = event["action"]
+    # `synchronize` action is when a new commit is made for the PR
     ignored_actions = set(("labeled", "synchronize"))
     if action in ignored_actions:
         msg = "Ignoring {action} events from github".format(action=action)
+        print(msg, file=sys.stderr)
         return msg, 200
 
+    _msg = "{}/pull/{} {}".format(repo, PR_NUMBER, action)
```

Comment:
An underscored local seems odd. And this isn't a message. Seems like "activity" to me.